examples/hello-world/index.js

```diff
@@ -5,7 +5,7 @@ var express = require('../../');
 var app = module.exports = express()
 
 app.get('/', function(req, res){
-  res.send('Hello World');
+  res.send('Hello World1');
```

Comment:
```suggestion
  res.send('Hello World!');
```

however it this pr really needed?